src/types/analytics_default_properties.ts

```diff
@@ -0,0 +1,17 @@
+type properties = {
```

Comment:
I think Mixpanel tracks the host automatically, right?

Reply:
Host added